data/test/scenarios/test_lua_name.cfg

```diff
@@ -0,0 +1,18 @@
+
+# Test that an error message in Lua includes the value of the tag's name key
+{GENERIC_UNIT_TEST test_lua_name (
+    [event]
+        name=start
+        [lua]
+            name=test
```

Comment:
Likely not needed, but I'd prefer the name here be something more unique than just "test".